src/_posts/2019-03-17-nature-in-code.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+asset-type: post
+name: nature-in-code
+title: Nature In Code
+date: 2019-03-17 23:00:00 +01:00
+author: Solange U. Gasengayire
+image:
+    src: /assets/custom/img/blog/2019-03-17-nature-in-code/nature-in-code.jpg
+abstract: How basic programming constructs can be used as a powerful tool to describe, understand and reason about our natural world.
+alias: [/2019/03/17/nature-in-code]
+
+---
+
+A little over two years ago, while exploring the edX platform for online courses, I came across a fantastic one about programming for beginners called [Nature in Code: Biology in JavaScript][1]. This course shows how basic programming constructs can be used as a powerful tool to describe, understand and reason about our natural world.
+
+In this post, we'll at how the aforementioned course teaches and translates scientific ideas such as evolution and epidemics, using simple models, into code. As a starting point, we define a `null` model, then we look into how mutation and migration (two of the main forces that lead to evolution) are implemented. And then, building from the previous ideas, we will briefly touch on how the spread of infectious diseases can be represented in code.
+
+## The Hardy-Weinberg Model
+
+In biology, **evolution** is the change in genetic composition of a population over time. The course identifies four forces that lead to this change:
+
+  - **natural selection**, as described by Charles Darwin, where <cite>individuals with characteristics best suited to their environment are more likely to survive, reproduce and pass their genes onto their children</cite>[<sup>1</sup>][2]
+  - **genetic drift**, where the change in genetic composition is due to random chance
+  - **migration**, which refers to population moving from one place to another
+  - **mutation**, which is described as the *ultimate engine of diversity creation*.
+
+The course introduces the **Hardy-Weinberg Model** - also known as the `null` model, as being the model that describes how a system would behave without any of the forces of interest.
+
+It relies on the following simplifying assumptions:
+
+- an infinite population size
+- non-overlapping generations
+- sexual reproduction happens randomly
+- and none of the four forces above is in action
+
+Let's consider a basic model of a gene declined into two alleles a<sub>1</sub> and a<sub>2</sub>. A **gene** is defined as <cite>the basic physical and functional unit of heredity</cite>.[<sup>2</sup>][3] Every gene exists in multiple versions, called **alleles**, and these versions are the ones that make us unique.
```

Comment:
Add ** markers on the first sentence to both gene and allele. That way is clear that they are something special. Specially the second one is important, as I don't think is a well enough word. Keep those markers on the rest of the paragraph, though.